spec/models/miq_ae_method_spec.rb

```diff
@@ -1,60 +1,46 @@
 describe MiqAeMethod do
-  before do
-    @user = FactoryGirl.create(:user_with_group)
-  end
-
+let (:user) { FactoryGirl.create(:user_with_group) }
```

Comment:
There shouldn't be a space between `let` and `(`

Reply:
Oh! I am trying, this is my first time with Ruby. There is a possibility that even after next update, there will still remain some mistakes.